src/webgpu/api/validation/createTexture.spec.ts

```diff
@@ -281,12 +281,17 @@ g.test('sampleCount,various_sampleCount_with_all_formats')
     const { format } = t.params;
     const info = kTextureFormatInfo[format];
     t.skipIfTextureFormatNotSupported(format);
+    t.selectDeviceForRenderableColorFormatOrSkipTestCase(format);
     t.selectDeviceOrSkipTestCase(info.feature);
   })
   .fn(t => {
     const { dimension, sampleCount, format } = t.params;
     const info = kTextureFormatInfo[format];
 
+    if (sampleCount > 1) {
```

Comment:
should this instead just move to the `success` calculation?

Reply:
You are right. Fixed